src/test/java/bio/terra/service/snapshotbuilder/SnapshotBuilderServiceTest.java

```diff
@@ -37,4 +40,38 @@ void createSnapshotRequest() {
             datasetId, SnapshotBuilderTestData.createSnapshotAccessRequest(), email),
         response);
   }
+
+  @Test
+  void enumerateSnapshotRequestsByDatasetId() {
```

Comment:
I'm having a hard time gauging the value of this test in its current form.  I think it's because it's trying to assert that our enumeration is populated by another method in the same service.

If `convertToEnumerateModel` were broken out into an intermediate class or another class that we could mock in this test, we could then mock the method and verify the response that way.

If not, I'd recommend the following:
- Make `SnapshotBuilderService.convertToEnumerateModel` private since it's only meant to be called within the service and not externally
- Only unit test `SnapshotBuilderService.enumerateByDatasetId` but in the way that you were previously testing `convertToEnumerateModel` -- that way your test isn't tied to the specific implementation, but rather the desired behavior of the public method

Reply:
I agree, I was just about to make the same comment; these two tests should be combined into one and it's OK to test `convertToEnumerateModel()` indirectly that way.